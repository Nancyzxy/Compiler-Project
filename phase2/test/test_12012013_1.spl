float inc(float n)
{
  return n;
}
int add(int b){
  return b;
}
int test_2_r02()
{
  float k = 4.5;
  int a = 0;
  int r = add(a) + inc(k);
  return a;
}