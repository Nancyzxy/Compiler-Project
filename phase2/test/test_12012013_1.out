Error type 7 at Line 12: binary operation on non-number variables
